src/app/create-contact/create-contact.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Contact, ContactService } from '../contact.service';
+import { Router } from '@angular/router';
+import * as uuid from 'uuid';
+@Component({
+  selector: 'app-create-contact',
+  templateUrl: './create-contact.component.html',
+  styleUrls: ['./create-contact.component.css'],
+})
+export class CreateContactComponent {
+  contactForm: FormGroup;
+
+  constructor(
+    private _formBuilder: FormBuilder,
```

Comment:
Be wary of using `_` in ts/js - it's typically used to denote unused variables. You might find linters may get confused/throw a warning if you use them (depending on how strict the linting is)